### PR TITLE
Added some extra books and websites in content

### PR DESCRIPTION
**Pull request by AK16092003**

### CODE REVIEW

File: content/books/books.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ## Books 📗
 
 - [Algorithms Jeff Erickson](https://jeffe.cs.illinois.edu/teaching/algorithms/book/Algorithms-JeffE.pdf) 
-- [Grokking Algorithms An illustrated guide for programmers and other curious people](https://www.manning.com/books/grokking-algorithms) 
\ No newline at end of file
+- [Grokking Algorithms An illustrated guide for programmers and other curious people](https://www.manning.com/books/grokking-algorithms) 
+- [Competitive Programmer's Handbook](https://cses.fi/book/book.pdf)
```
Comment (SamuelBFavarin) [inline content/books/books.md:4]: Please, can you follow the alphabetical sort? Thanks!

File: content/websites/websites.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
 ## Website 👨‍💻
 
-- [Hackerhank](https://www.hackerrank.com/)
```
Comment (SamuelBFavarin) [inline content/websites/websites.md:3]: Same here!

**Review by SamuelBFavarin — CHANGES_REQUESTED** [review]
Hi @AK16092003 thank you for joining and contributing to this repo! I left some comments about following the alphabet sorting in content `.md` files.

**Comment by AK16092003** [comment]
@SamuelBFavarin 
i have done the changes , thank you

**Review by SamuelBFavarin — APPROVED** [review]
LGTM, thanks! 🚀

**Pull request merged**